### ISSUE TITLE
Support Twitch's new animated emotes

### ISSUE THREAD

**Issue by karl-police** [Chatterino/chatterino2#2897]
Twitch added animated emotes, however they're yet not animating in Chatterino.

**Comment by pajlada** [comment]
Works for me in...
 - The emote popup
 - The emote colon dialog
 - Chat

Please make sure you're on the latest Nightly (or a version later than https://github.com/Chatterino/chatterino2/commit/6611c24e1375380037a1aaff94b2709d8d5f827a )

**Comment by karl-police** [comment]
> Works for me in...
> 
> * The emote popup
> * The emote colon dialog
> * Chat
> 
> Please make sure you're on the latest Nightly (or a version later than [6611c24](https://github.com/Chatterino/chatterino2/commit/6611c24e1375380037a1aaff94b2709d8d5f827a) )

How can I force an update without reinstalling the app?

**Comment by zneix** [comment]
> > Works for me in...
> > 
> > * The emote popup
> > * The emote colon dialog
> > * Chat
> > 
> > Please make sure you're on the latest Nightly (or a version later than [6611c24](https://github.com/Chatterino/chatterino2/commit/6611c24e1375380037a1aaff94b2709d8d5f827a) )
> 
> How can I force an update without reinstalling the app?

You can't. Nightly has to be downloaded and extracted manually.

**Comment by karl-police** [comment]
> > > Works for me in...
> > > 
> > > * The emote popup
> > > * The emote colon dialog
> > > * Chat
> > > 
> > > Please make sure you're on the latest Nightly (or a version later than [6611c24](https://github.com/Chatterino/chatterino2/commit/6611c24e1375380037a1aaff94b2709d8d5f827a) )
> > 
> > 
> > How can I force an update without reinstalling the app?
> 
> You can't. Nightly has to be downloaded and extracted manually.

Why is it called "Nightly", I've seen names like that in apps before.

**Comment by zneix** [comment]
> Why is it called "Nightly", I've seen names like that in apps before.

tl;dr: It's basically an automated build made by CI which is a product of testing if app builds successfully on checks. See [a random article I've quickly found](https://softwareengineering.stackexchange.com/questions/56490/what-does-nightly-builds-mean) for a better explanation.
